src/context/authentication/firebase-dao.ts

```diff
@@ -46,9 +50,9 @@ const authenticationFirebaseDao = (
       .catch((e): Err<Error, void> => err(e, undefined));
   };
 
-  const get = (): Result<Error, string> => {
+  const get = (): Result<Error, AuthenticationUser> => {
     if (user) {
-      return ok(user.uid);
+      return ok({ id: user.uid, hasVerifiedEmail: user.emailVerified }: AuthenticationUser);
     }
     return err(new Error('user not logged in'), '');
```

Comment:
We will need to create a default AuthenticationUser and change this to:
`return err(new Error('user not logged in'), defaultAuthenticationUser);`
Or something like that.